### PR TITLE
Arranged ipsec info

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://trello.com/c/AjbciweI/589-1-update-ipsec-documentation-to-account-for-newer-libreswan

@dcbw PTAL 

Took away the "At this time, version 3.15..." sentence and arranged and reworded some of the sections.

### CODE REVIEW

**Comment by dcbw** [comment]
/lgtm

Also re-tested opportunistic IPSec support with the docker-in-docker infrastructure and it appears to work.

**Comment by bfallonf** [comment]
Big thanks @dcbw !

@openshift/team-documentation PTAL. I edited out the reference to the "at this time" and reworded some of the info.

**Comment by mburke5678** [comment]
@bfallonf One nit. LGTM otherwise

File: admin_guide/ipsec.adoc [hunk 1]
```diff
@@ -37,21 +37,21 @@ with a host that does.
...
-IPSec and the SDN encapsulation.
-
+* See the
+ xref:../install_config/configuring_sdn.adoc#configuring-the-pod-network-on-nodes[Configure
+ the pod network on nodes] section for information on how to configure the MTU
+ to allow space for the IPSec header. This topic describes an IPSec
+ configuration that requires 62 bytes. If the cluster is operating on an
+ ethernet network with an MTU of 1500 then the SDN MTU should be 1388, to allow
+ for the overhead of IPSec and the SDN encapsulation.
++
```
Comment (mburke5678) [inline admin_guide/ipsec.adoc]: I think ethernet should be initial-capped: Ethernet. Style guide says to cap it; but our docs are inconsistent. 7 instances capped; 8 lower-case.
https://mojo.redhat.com/docs/DOC-28115#e

Comment (bfallonf) [reply admin_guide/ipsec.adoc]: @mburke5678 Checks out:

https://www.reddit.com/r/grammar/comments/34eldl/capitalize_ethernet/

To stick with IBM I'll capatalize. Good eye!

**Comment by bfallonf** [comment]
Accidentally deleted the wrong origin branch and now this got deleted. Reopened at #7872

**Pull request closed without merging**